pkg/cue/model/sets/operation.go

```diff
@@ -115,7 +115,7 @@ func listMergeProcess(field *ast.Field, key string, baseList, patchList *ast.Lis
 			kmaps[fmt.Sprintf(key, blit.Value)] = patchList.Elts[i]
 		}
 		if !foundPatch {
-			return
+			continue
```

Comment:
Can you add a test in application_controller?

Reply:
An e2e test is added.

Reply:
UT is also added.